frontend/src/components/header/header.graphql

```diff
@@ -1,23 +1,45 @@
-fragment Link on MenuLink {
-  title(language: $language)
-  href(language: $language)
-
-  page {
-    slug(language: $language)
-  }
-}
-
-query Header($code: String!, $language: String!) {
+query Header($code: String!) {
   conference(code: $code) {
     id
-    conferenceMenu: menu(identifier: "conference-nav") {
+    conferenceMenuEn: menu(identifier: "conference-nav") {
```

Comment:
this allows Apollo to reuse the query result and not have to fetch the menus every time we switch language